allennlp/modules/stacked_elmo_lstm.py

```diff
@@ -0,0 +1,146 @@
+"""
+A stacked bidirectional LSTM with skip connections between layers.
+"""
+
+from typing import Optional, Tuple
+import torch
+from torch.nn.utils.rnn import PackedSequence, pad_packed_sequence
+from allennlp.modules.elmo_lstm_cell import ElmoLstmCell
+from allennlp.common.checks import ConfigurationError
+
+
+class ElmoLstm(torch.nn.Module):
+    """
+    A stacked, bidirectional LSTM which uses :class:`~allennlp.modules.elmo_lstm_cell.ElmoLstmCell` 's
+    with highway layers between the inputs to layers.
+    The inputs to the forward and backward directions are independent - forward and backward
+    states are not concatenated between layers.
+
+    Parameters
+    ----------
+    input_size : ``int``, required
+        The dimension of the inputs to the LSTM.
+    hidden_size : ``int``, required
+        The dimension of the outputs of the LSTM.
+    cell_size : ``int``, required.
+        The cell dimension of the :class:~`allennlp.modules.elmo_lstm_cell.ElmoLstmCell`.
+    num_layers : ``int``, required
+        The number of stacked LSTMs to use.
+    recurrent_dropout_probability: ``float``, optional (default = 0.0)
+        The dropout probability to be used in a dropout scheme as stated in
+        `A Theoretically Grounded Application of Dropout in Recurrent Neural Networks
+        <https://arxiv.org/abs/1512.05287>`_ .
+    state_projection_clip_value: ``float``, optional, (default = 3)
+        The magnitude with which to clip the hidden_state after projecting it.
+    memory_cell_clip_value: ``float``, optional, (default = 3)
+        The magnitude with which to clip the memory cell.
+    """
+    def __init__(self,
```

Comment:
It'd be easier to maintain if you just add `num_layers` then relegate the rest to `**kwargs` that are passed down the LSTMCell.  E.g. you also need to modify the default behavior of `state_projection_clip_value` here too.  Or imagine adding an option to the LSTMCell, you will also need to modify this code too.

I'd even tie them together more closely and move all of the cell related code into this file to make the connection explicit.  I find there is cognitive drain to switch between files for closely related code.